### PR TITLE
docs: Update "How to test parcels and preview scenes" section

### LINKED ISSUE #551: The flow for loading preview scenes in Unity is broken
These instructions no longer work:
https://github.com/decentraland/explorer/blob/master/docs/how-to-test-parcels-and-preview-scenes.md#unity-editor-debugging-with-dcl-scene-in-preview-mode

The data is never sent to Unity, there are some errors in the browser console:

A discord user was asking about this and when I tried it locally before answering himI saw that the flow is broken.

When we fix this we could also update the URL and remove the `UNITY_ENABLED=true` param

### PR DESCRIPTION
**Pull request by sandrade-dcl**
# What? 
Fix decentraland/unity-renderer/issues/551

Update the "How to test parcels and preview scenes" section.

# Why? 
The processes described in the documentation were out of date.

### CODE REVIEW

File: docs/how-to-test-parcels-and-preview-scenes.md [hunk 1]
```diff
@@ -20,28 +20,57 @@ Note that the test-scenes building may take a while (10 mins or more).
 
 4. To spawn at the specific coordinates of a test scene append the position query paramter:
 
-    http://localhost:3000/?DEBUG_MODE&position=-100,100
+    http://localhost:3000/?DEBUG_MODE&position=-1,27
```
Comment (D4rWiNSS) [inline docs/how-to-test-parcels-and-preview-scenes.md:23]: Is this change necessary?

**Review by D4rWiNSS — APPROVED** [review]
I don't know if this works, but english wise LGTM!

File: docs/how-to-test-parcels-and-preview-scenes.md (hunk 1 above)
Comment (sandrade-dcl) [reply docs/how-to-test-parcels-and-preview-scenes.md:23]: I wanted to change these coords because the `-100,100` was giving me an error (probably because this test scene is broken) and I think it is more proper to give the users a valid example.

Comment (menduz) [reply docs/how-to-test-parcels-and-preview-scenes.md:23]: I'm curious about the broken scene. Does it throw an exception? Does it load at all?

**Review by BrianAmadori — APPROVED** [review]

File: docs/how-to-test-parcels-and-preview-scenes.md (hunk 1 above)
Comment (sandrade-dcl) [reply docs/how-to-test-parcels-and-preview-scenes.md:23]: The problem seems to be specifically in the test scene `-102,102`. I attach the errors throwed in the browser console:

image
image

Comment (menduz) [reply docs/how-to-test-parcels-and-preview-scenes.md:23]: oh ok, that is not good. It seems like scenes can break the renderer. Could you track it as critical? People could not go to nearby locations CC @olavra . I left my approval for the docs, that should be handled in another pr

**Review by menduz — APPROVED** [review]

File: docs/how-to-test-parcels-and-preview-scenes.md (hunk 1 above)
Comment (sandrade-dcl) [reply docs/how-to-test-parcels-and-preview-scenes.md:23]: Ok, we're going to create a task for this 👍
